This patch fixes error:

    env XFORM_USE_PRECOMP=xsrc/precomp.h ../racketcgc  -cqu ./xform.rkt --setup . --depends --cpp "gcc -E -I./.. -I./../include -I./../../rktio -I../../rktio -Os -fomit-frame-pointer -pthread  -DUSE_SENORA_GC   -DMZ_USES_SHARED_LIB "  --keep-lines -o xsrc/future.c ./../src/future.c
    Error [GCING] 234 in ./../src/bool.c: Function mz_long_double_eqv declared __xform_nongcing__, but includes a function call at __FLOAT_BITS.
    Error [GCING] 270 in ./../src/bool.c: Function double_eqv declared __xform_nongcing__, but includes a function call at __FLOAT_BITS.
    xform: Errors converting
      context...:
       src/racket/gc2/xform-collects/compiler/priva...:8:2: xform
       src/racket/gc2/xform-collects/xform/xform-mo...: [running body]
       src/racket/gc2/xform.rkt: [running body]

Resources:
  * https://github.com/racket/racket/issues/1638
  * https://github.com/NixOS/nixpkgs/pull/31017
  * https://github.com/racket/racket/pull/1886

--- a/collects/compiler/private/xform.rkt
--- a/collects/compiler/private/xform.rkt
@@ -924,6 +924,7 @@
                |GetStdHandle| |__CFStringMakeConstantString|
                _vswprintf_c
 	       malloc strdup
+	       __FLOAT_BITS __DOUBLE_BITS ; musl
                
                scheme_make_small_bignum scheme_make_small_rational scheme_make_small_complex))
         (define non-functions-table
